CPAC/pipeline/engine.py

```diff
@@ -2511,6 +2514,18 @@ def ingress_pipeconfig_paths(cfg, rpool, unique_id, creds_path=None):
                 "",
                 f"{key}_config_ingress",
             )
+        #check if val is a nifti file .nii.gz
+        if val.endswith('.nii.gz'):
```

Comment:
```suggestion
        # check if val is a nifti file .nii.gz
        if val.endswith(".nii.gz"):
```

[pre-commit](https://github.com/FCP-INDI/C-PAC/blob/dbdacd1c6a745b4847b2c6abdf2c2f3d6150511f/CONTRIBUTING.md#pre-commit) should have caught this?